final_prep/README.md

```diff
@@ -19,27 +19,33 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [👍🏾] I have refactored my `section1` exercises to the best of my ability
+- [👍🏾] I have refactored my `section2` exercises to the best of my ability
+- [👍🏾] I have refactored my `section3` exercises to the best of my ability
+- [👍🏾] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [👍🏾] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [👍🏾] Study/Project work time
+- [👍🏾] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+- ![alt text](https://user-images.githubusercontent.com/79548116/116721018-7654d800-a9a2-11eb-8c03-42530d6ce9ae.png "Logo Title Text 1")
+
 - `Add Week 2 Screenshot Here`
+- ![alt text](https://user-images.githubusercontent.com/79548116/116721229-b320cf00-a9a2-11eb-81fe-4de468c9c53c.png "Logo Title Text 1") 
+
 - `Add Week 3 Screenshot Here`
+- ![alt text](https://user-images.githubusercontent.com/79548116/116721421-e6635e00-a9a2-11eb-864e-ecab4a7d1d91.png "Logo Title Text 1")
+
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: https://gist.github.com/karanm645/8c437bc026fa6889220c84281ebc0fc9
```

Comment:
Nice job here. In your message to your mentor, I'd include some detail about what you did before deciding to come to Turing, as well as what specifically excites you about making this change.

Reply:
Finished

Reply:
done